strings/owo_talk.json

```diff
@@ -23,10 +23,10 @@
         "person": "cat",
         "lizard": "scalie",
         "damn": "mew",
-        "kill": "yiff",
-        "killed": "yiffed",
-        "murder": "yiff",
-        "murdered": "yiffed",
+        "kill": "keel",
+        "killed": "keeled",
+        "murder": "mworder",
+        "murdered": "mworder",
```

Comment:
```suggestion
        "kill": "nuzzle",
        "killed": "nuzzled",
        "murder": "nuzzle",
        "murdered": "nuzzled",
```

Reply:
Fine. If it leads to my admintools getting fixed quicker.